reference/platform-app-yaml.md

```diff
@@ -188,7 +188,7 @@ The `locations` key allows you to provide specific parameters for different URL
 It has a few subkeys, which are:
 
 * `root`: The folder to serve static assets for this location from relative to the application root. The application root is the directory in which the `.platform.app.yaml` file is located.  Typical values for this property include `public` or `web`.  Setting it to `"/"` is not recommended, and its behavior may vary depending on the type of application.
-* `passthru`: Whether to forward disallowed and missing resources from this location to the application. Can be true, false or a URI path string relative to the web root. For non-PHP applications it will typically be just true or false.  In a PHP application, this will typically be the front controller such as `/index.php` or `/app.php`.  If your application relies on a `.htaccess` file with `mod_rewrite` under Apache, this entry is the equivalent. Note: The value of `passthru` is relative to the domain, which means if it does not begin with the same value as the location key it is under the passthru may evaluate to another entry. That may be useful when you want different cache settings for different paths, for instance, but want missing files in all of them to map back to the same front controller.
+* `passthru`: Whether to forward disallowed and missing resources from this location to the application. Can be true, false or a URI path string relative to the web root. For non-PHP applications it will generally be just true or false.  In a PHP application, this will typically be the front controller such as `/index.php` or `/app.php`.  If your application relies on a `.htaccess` file with `mod_rewrite` under Apache, this entry is the equivalent. Note: The value of `passthru` must be an absolute URL path, which means if it does not begin with the same value as the location key it is under the passthru may evaluate to another entry. That may be useful when you want different cache settings for different paths, for instance, but want missing files in all of them to map back to the same front controller.
```

Comment:
Maybe something like this then
> Note: if setting the `passthru` as a URL path, make sure it is absolute, i.e. begins with a `/`

because if it looks like an absolute path it should be fairly clear what it means, right?